packages.dhall

```diff
@@ -105,8 +105,8 @@ in  upstream
 -------------------------------
 -}
 let upstream =
-      https://github.com/purescript/package-sets/releases/download/psc-0.14.5-20211116/packages.dhall
-        sha256:7ba810597a275e43c83411d2ab0d4b3c54d0b551436f4b1632e9ff3eb62e327a
+      https://github.com/purescript/package-sets/releases/download/psc-0.14.5-20220224/packages.dhall
+        sha256:67cc3d4f0e8fb72bb1413ba94ddd72a3ceb0783eb725e3b22ad7568b3b581163
```

Comment:
I think we should revert this change, but in case we keep it I wanted to say we should make sure the templates upstream is set to the same

Reply:
I went through our spago-packages.nix and the diff between the two package sets, these are the only differences we can observe:
```diff
   , repo = "https://github.com/purescript/purescript-free.git"
-  , version = "v6.1.0"
+  , version = "v6.2.0"
   , repo = "https://github.com/purescript-node/purescript-node-fs.git"
-  , version = "v6.1.0"
+  , version = "v6.2.0"
   , repo = "https://github.com/purescript-contrib/purescript-parsing.git"
-  , version = "v7.0.1"
+  , version = "v8.2.0"
   , repo = "https://github.com/purescript-contrib/purescript-unicode.git"
-  , version = "v5.0.0"
+  , version = "v5.0.1"
```

Reply:
It was purescript-parsing. Downgrading

Reply:
Thought so - annoyingly there was no breaking changes between those versions, only bug fixes. So it's odd that something broke

Reply:
I think the culprit is 

>     Parser.String.eof Set consumed on success so that this parser combines correctly with notFollowedBy eof. Added a test for this. (#140 by @jamesdbrock)

https://github.com/purescript-contrib/purescript-parsing/blob/main/CHANGELOG.md